scrapy/spiderloader.py

```diff
@@ -33,12 +41,15 @@ def _check_name_duplicates(self):
             msg = ("There are several spiders with the same name:\n\n"
                    "{}\n\n  This can cause unexpected behavior.".format(
                         "\n\n".join(dupes)))
-            warnings.warn(msg, UserWarning)
+            warn(msg, UserWarning)
```

Comment:
💄 I usually prefer warning.warn, because otherwise it is less clear if that's logging.warn or warning.warn.